src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
The Main class should not start with an empty line. Please remove the empty line at the beginning of the class definition.